src/components/ArtistPieChart/ArtistPieChart.scss

```diff
@@ -0,0 +1,230 @@
+.container {
+    margin-left: 5px;
+    width: 600px;
+    height: 600px;
+    background-color:rgba(0, 0, 0, 0.24);
+    display: flex;
+    flex-direction: row;
+    position: relative;
+    color: black;
+    border-color: black;
+    box-shadow: 1px 1px 12px black;
+}
+
+.column {
+    width: 50%;
+    height: 100%;
+}
+
+.row {
+    height: 50%;
+    display: flex;
+    flex-direction: column;
+}
+
+.quarter-circle-top-left{
+    width: 50%;
+    height: 50%;
+    margin-bottom: auto;
+    background: #720D6D;
+    border-radius: 0 0 100% 0;
+    -moz-border-radius: 0 0 100% 0;
+    -webkit-border-radius: 0 0 100% 0;
+    overflow: hidden;
+    transition: width 0.5s, height 0.5s;
+    &:hover {
+        width: 53%;
+        height: 53%;
+     }
+     z-index: 1;
+}
+
+.quarter-circle-top-right{
```

Comment:
I see majority of stuff here repeated. cant we have the common properties in a separate class and have two classes asociated to the respective html element rather than one